verification/dependencies/strings/strings.gobra

```diff
@@ -158,7 +160,7 @@ func TrimPrefix(s, prefix string) string
 // TrimSuffix returns s without the provided trailing suffix string.
 // If s doesn't end with suffix, s is returned unchanged.
 decreases _
-func TrimSuffix(s, suffix string) string
+pure func TrimSuffix(s, suffix string) string
```

Comment:
why did you make this pure?

Reply:
I would prefer adding the explicit contract, as otherwise it may not be clear why this is marked as pure